modules/nw-sriov-dual-nic-proc.adoc

```diff
@@ -0,0 +1,102 @@
+// Module included in the following assemblies:
+//
+// * networking/hardware_networks/configuring-sriov-device.adoc
+
+:_content-type: CONCEPT
+[id="nw-dual-nic-proc_{context}"]
+= Setting up networking for dual-nic on day-1
+
+== Agent based installer
```

Comment:
Where is this module included? I couldn't find it in the docs preview pages.

Reply:
@cgoncalves sorry my bad let me remove this file from the PR